### PR TITLE
php

### PR DESCRIPTION
**Pull request by hzoo**
Ref https://github.com/babel/babel.github.io/issues/1270

Just copied the current html/js for the repl and removed the unnecessary options

screen shot 2017-07-11 at 11 59 58 am

Ok unfortunately I don't really know php but it seems to work 😅

In terms of "patching" could write a polyfill for things

### CODE REVIEW

File: php.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+layout: default
+permalink: /php/
+fullscreen: true
+redirect_from: "/php.html"
+custom_js_with_timestamps:
+- pretty-format.js
+- php.js
+third_party_js:
+- https://unpkg.com/babel-standalone@6/babel.min.js
+- https://packd.now.sh/babel-preset-php@1.0.0
```
Comment (hzoo) [inline php.html]: ^

File: scripts/php.js [hunk 2]
```diff
@@ -0,0 +1,412 @@
+// polyfill
+window.echo = console.log.bind(console);
```
Comment (hzoo) [inline scripts/php.js]: basic polyfill

File: scripts/php.js [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+window.global = window;
+
+(function(babel, $, _, ace, LZString, window) {
+  'use strict';
+  var UPDATE_DELAY = 500;
+
+  Babel.registerPreset('php', babelPresetPhp);
```
Comment (hzoo) [inline scripts/php.js]: register it

**Comment by ericnkatz** [comment]
Sidenote, this preset is pretty cool. hah.

**Pull request merged**